keyboards/dp60/matrix.c

```diff
@@ -0,0 +1,245 @@
+#include "quantum.h"
```

Comment:
Is there a reason for the custom matrix?

Reply:
This pcb reserved ports for connecting an Adafruit SPI BLE friend, so have to use two 74ls138 for column port strobing. This was why it have to use custom matrix.

Reply:
aah, okay.

Reply:
Well, could this be updated to use the debouncing API? 

A la
https://github.com/qmk/qmk_firmware/blob/b624f32f944acdc59dcb130674c09090c5c404cb/quantum/matrix.c#L244-L278

Reply:
Good catch on the EEPROM addresses that's probably why I was seeing some strangeness. I'll update the matrix strobing this weekend!

Reply:
integrating the matrix strobing is going to take a little more thought as a simple replace actually breaks the switch matrix =(. This is probably a task for another bug.